jwst/mrs_imatch/mrs_imatch_step.py

```diff
@@ -29,14 +20,27 @@ class MRSIMatchStep(Step):
         bkg_degree = integer(min=0, default=1) # Degree of the polynomial for background fitting
         subtract = boolean(default=False) # subtract computed sky from 'images' cube data?
         skip = boolean(default=True) # Step must be turned on by parameter reference or user
-    """ # noqa: E501
+    """  # noqa: E501
 
     reference_file_types: list = []
 
     def process(self, images):
+        """
+        Run this step for a given list of images.
 
+        Parameters
+        ----------
+        images : file path or list of DataModels
+            If a file path, initialize from an association table.
+            If a list, can be a list of DataModels of any type
+
+        Returns
+        -------
+        images : file path or list of DataModels
```

Comment:
I don't think the output would ever be a file path, I think it's always `ModelContainer`?

edit: I now see that it just returns `images` with no modifications.  Which doesn't make any sense to me... I know the step is deprecated, though, so maybe it need not make any sense.

I think it would be good to add somewhere in the documentation of the step that it is deprecated.

Reply:
Same as https://github.com/spacetelescope/jwst/pull/9474#discussion_r2098754038

Reply:
> I now see that it just returns `images` with no modifications.

That is my understanding as well from quick glance at the code unless `ModelContainer(images)` changes the input in-place, which also could be dangerous. 🤷‍♀️